components/mosart/src/cpl/rof_comp_esmf.F90

```diff
@@ -676,16 +677,24 @@ subroutine rof_import_esmf( x2r_array, rc)
 
     nliq = 0
     nfrz = 0
+    nmud = 0
+    nsan = 0
     do nt = 1,nt_rtm
        if (trim(rtm_tracers(nt)) == 'LIQ') then
           nliq = nt
        endif
        if (trim(rtm_tracers(nt)) == 'ICE') then
           nfrz = nt
        endif
+       if (trim(rtm_tracers(nt)) == 'MUD') then
+          nmud = nt
+       endif
+       if (trim(rtm_tracers(nt)) == 'SAN') then
+          nmud = nt
+       endif
     enddo
-    if (nliq == 0 .or. nfrz == 0) then
-       write(iulog,*) sub,': ERROR in rtm_tracers LIQ ICE ',nliq,nfrz,rtm_tracers
+    if (nliq == 0 .or. nfrz == 0 .or. nmud == 0 .or. nsan == 0) then
```

Comment:
I suggest splitting this long if into four ifs.

Reply:
I followed the original style in this line. It was "if (nliq == 0 .or. nfrz == 0) then" for two tracers. I just expanded it since we now have four tracers. I feel we'd better keep it this way.

Reply:
See my comment below